CSS/see_article.css

```diff
@@ -112,56 +161,276 @@ button a:focus {
     }
 }
 
-@media(max-width: 850px) {
+
+@media(min-width: 901px) {
+    #box {
+        display: grid;
```

Comment:
it seems that a lot of the properties of the box are the same amongst the different media queries. might need a refactor.
Some other elements also seems to have duplicated code in them.

Reply:
Has been corrected yesterday.